lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -172,6 +161,53 @@ def self.comment_on_pr(project_slug:, pr_number:, body:, reuse_identifier: Secur
 
         reuse_identifier
       end
+
+      # Update a milestone for a repository
+      #
+      # @param [String] repository The repository name (including the organization)
+      # @param [String] number The number of the milestone we want to fetch
+      # @param options [Hash] A customizable set of options.
+      # @option options [String] :title A unique title.
+      # @option options [String] :state
+      # @option options [String] :description A meaningful description
+      # @option options [Time] :due_on Set if the milestone has a due date
+      # @return [Milestone] A single milestone object
+      #
+      def self.update_milestone(repository:, number:, options: {})
+        client.update_milestone(repository, number, options)
+      end
+
+      # Unlock a single branch from a repository
+      #
+      # @param [String] repository The repository name (including the organization)
+      # @param [String] number The branch name
+      #
+      def self.remove_branch_protection(repository:, branch:, options:)
+        client.unprotect_branch(repository, branch_name, options)
+      end
+
+      # Lock a single branch from a repository
+      #
+      # @param [String] repository The repository name (including the organization)
+      # @param [String] number The branch name
+      #
+      def self.set_branch_protection(repository:, branch:, options:)
+        client.protect_branch(repository, branch_name, options)
+      end
+
+      # Creates a GithubToken Fastlane ConfigItem
+      #
+      # @return [FastlaneCore::ConfigItem] The Fastlane ConfigItem for GitHub OAuth access token
+      #
+      def self.github_token_config_item
+        return FastlaneCore::ConfigItem.new(
```

Comment:
💄  Nit: In Ruby, the `return` statement is implicit on the last statement of a function (similar to how you can omit `return` in Swift for one-liner functions), and code style conventions suggest to remove it (which `rubocop` would probably have caught if I had re-triggered CI on your fork, but I haven't yet since your latest commits).

See https://docs.rubocop.org/rubocop/cops_style.html#styleredundantreturn

Reply:
Thanks for pointing this out! I'm so used to putting the `return` on functions that when it didn't raise an error I assume that was correct 😅.